File: Team 6367 - The ElectroLights - 2018 Robot/src/org/usfirst/frc/team6367/robot/io/DriverInput.java
package org.usfirst.frc.team6367.robot.io;

import edu.wpi.first.wpilibj.Joystick;

public class DriverInput {

	private Joystick driverStick;
	private Joystick mechanismStick;
	
	
	public DriverInput() {
		driverStick = new Joystick(0);
		mechanismStick = new Joystick(1);
	}
	
	public Joystick getDriverStick() {
		return driverStick;
	}
	
	public Joystick getMechanismStick() {
		return mechanismStick;
	}
	
}

File: Team 6367 - The ElectroLights - 2018 Robot/py/io/driverinput.py
import wpilib

class DriverInput:
    
    def __init__(self):
        self.driverStick = wpilib.Joystick(0)
        self.mechanismStick = wpilib.Joystick(1)
    
    def getDriverStick(self) -> wpilib.Joystick:
        return self.driverStick
    
    def getMechanismStick(self) -> wpilib.Joystick:
        return self.mechanismStick
